### ISSUE TITLE
Surveys.getAvailableSurveys() returns an array of null values on iOS

### ISSUE THREAD

**Issue by tedcurrent** [Instabug/Instabug-React-Native#307]
### Steps to Reproduce the Problem
1. Create a new survey
2. Call `Surveys.getAvailableSurveys(surveys => console.log(surveys))`

### Expected Behavior
Should return an array of surveys

### Actual Behavior
Returns an array of null values

### Instabug integration code
```
const initInstabug = () => {
  Instabug.startWithToken(EnvConfig.INSTABUG_TOKEN, [
    EnvConfig.INSTABUG_SHOW_BUTTON
      ? Instabug.invocationEvent.floatingButton
      : Instabug.invocationEvent.none
  ])

  const deviceId = `${DeviceInfo.getUniqueID()}::${DeviceInfo.getModel()}`

  Instabug.setUserAttribute('Device Identifier', deviceId || 'unknown')

  Instabug.setWelcomeMessageMode(Instabug.welcomeMessageMode.disabled)

  if (!EnvConfig.INSTABUG_SHOW_BUTTON) {
    Instabug.setEnabledAttachmentTypes(false, false, false, false)
  }

  Chats.setEnabled(false)
  BugReporting.setEnabled(true)
  Replies.setEnabled(true)

  Surveys.setEnabled(true)
  Surveys.setAutoShowingEnabled(false)

  Surveys.getAvailableSurveys(surveys => {
    console.warn(surveys)
  })
}
```
### SDK Version

`"instabug-reactnative": "^8.4.0"`

### React Native, iOS and Android Versions
`iOS 12.3.1`
`React Native 0.59.9`

**Comment by salmamali** [comment]
@tedcurrent Thanks for pointing this out. The fix will be included in the next release.

**Comment by salmamali** [comment]
@tedcurrent fixed in version 8.4.1
